apps/src/p5lab/P5Lab.js

```diff
@@ -272,6 +272,7 @@ P5Lab.prototype.init = function(config) {
 
   config.dropletConfig = dropletConfig;
   config.appMsg = this.isSpritelab ? spritelabMsg : gamelabMsg;
+  this.studioApp_.loadLibraryBlocks(config);
```

Comment:
Why is this needed? Aren't libraries just in App Lab?

Reply:
App Lab and Game Lab